### PR TITLE
Add chat categorization dataset

### PR DESCRIPTION
**Pull request by hinthornw**
This task is meant to test a couple things:
1. Classification -> both on common things where it is expected to perform well (e.g., sentiment, toxicity -> which currently is always 0)
2. Structured json output -> the schema is nested, which confused some of the smaller 7b models i tested out but works fine for llama 32b code instruct (and OAI/anthropic)

Includes a couple common things like enums.

TODO: Clean up notebooks and add more analysis

### CODE REVIEW

File: langchain_benchmarks/extraction/tasks/chat_extraction/schema.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    response_type_if_other: Optional[str] = None
+    confidence_level: int = Field(
+        ge=0, lt=6, description="The confidence of the assistant in its answer."
+    )
+    followup_actions: Optional[List[str]] = Field(
+        description="Actions the assistant recommended the user take."
+    )
+
+
+class generateTicket(BaseModel):
```
Comment (eyurtsev) [inline langchain_benchmarks/extraction/tasks/chat_extraction/schema.py:88]: Intentional camel case?

**Review by eyurtsev — APPROVED** [review]

File: langchain_benchmarks/extraction/tasks/chat_extraction/__init__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+from langchain.prompts import ChatPromptTemplate as C
```
Comment (eyurtsev) [inline langchain_benchmarks/extraction/tasks/chat_extraction/__init__.py:1]: Imo shouldn't use abbreviations here, ide should auto complete to save typing

File: langchain_benchmarks/extraction/tasks/chat_extraction/schema.py (hunk 1 above)
Comment (hinthornw) [reply langchain_benchmarks/extraction/tasks/chat_extraction/schema.py:88]: Ya but I can update. Always feels weird to capital case a function

File: langchain_benchmarks/extraction/tasks/chat_extraction/__init__.py (hunk 2 above)
Comment (hinthornw) [reply langchain_benchmarks/extraction/tasks/chat_extraction/__init__.py:1]: Works for me!

**Pull request merged**